src/store/mod.rs

```diff
@@ -134,6 +134,22 @@ impl Store {
             .map(|_| ())
             .map_err(StoreError::QueryError)
     }
+
+    pub fn remove_account(&self, account_id: u64) -> Result<(), StoreError> {
```

Comment:
Let's make sure related objects are deleted as well for this command. In the future this can be optional if we see a scenario where it's positive to eliminate only partial data related to an account.